docs/DeveloperGuide.md

```diff
@@ -378,234 +375,353 @@ Use case ends.
+1. User finds the index of the customer in the customer list whose orders they want to view.
+2. User inputs command to list all orders under the index of customer.
+3. SellSavvy displays the orders under the specified customer.
 
 Use case ends.
 
 **Extensions**
 
-* 3a. SellSavvy detects that there are no customers with the specified index.
-    * 3a1. SellSavvy displays an error that the customer index is invalid.
+* 2a. SellSavvy detects the customer index is missing or non-positive.
+    * 2a1. SellSavvy displays an error about the format and states the command format.
+
+  Use case ends.
+
+
+* 2b. SellSavvy detects that there are no customers with the specified index.
+    * 2b1. SellSavvy displays an error that the customer index is invalid.
 
     Use case ends.
 
-**Use case 7: Mark Order as Completed**
+**Use case 8: Mark Order as Completed**
 
-* **System**: SellSavvy
-* **Actor**: User
-* **Use Case**: UC07 - Mark Order as Completed
-* **Preconditions**: A customer's list of orders is being displayed.
+* **Use Case**: UC08 - Mark Order as Completed
+* **Preconditions**: There are orders displayed in the customer's list.
 * **Guarantees**:
     * Specified order will be marked as “Completed” if the input parameters are valid.
 
 **MSS**
 
-1. User wants to mark an order by a customer as completed.
-2. User finds the order they want to mark as completed.
-3. User specifies the index of the order.
-4. SellSavvy updates the status of the order and indicates that the action is successful.
+1. User finds the order they want to mark as completed.
+2. User specifies the index of the order.
+3. SellSavvy updates the status of the order and indicates that the action is successful.
 
 Use case ends.
 
 **Extensions**
 
-* 3a.  The specified order is already marked as "Completed".
-    * 3a1. SellSavvy displays a message stating that the order is already marked as completed.
+* 2a. SellSavvy detects that the order index is missing or non-positive.
+    * 2a1. SellSavvy displays an error about the format and states the command format.
+
+  Use case ends.
+
+
+* 2b.  The specified order is already marked as "Completed".
+    * 2b1. SellSavvy displays a message stating that the order is already marked as completed.
 
     Use case ends.
 
 
-* 3b. There are no orders with the specified index.
-  * 3b1. SellSavvy displays an error that the order index is invalid.
+* 2c. There are no orders with the specified index.
+  * 2c1. SellSavvy displays an error that the order index is invalid.
 
   Use case ends.
 
-**Use case 8: Remove "Completed" Marking from Order**
+**Use case 9: Remove "Completed" Marking from Order**
 
-* **System**: SellSavvy
-* **Actor**: User
-* **Use Case**: UC08 - Remove "Completed" Marking from Order
-* **Preconditions**: A customer's list of orders is being displayed.
+* **Use Case**: UC09 - Remove "Completed" Marking from Order
+* **Preconditions**: There are orders displayed in the customer's list.
```

Comment:
same issue as markorder

Reply:
Changing to `A customer's list of order's is being displayed.`